index.html

```diff
@@ -91,8 +90,200 @@ <h1>I'm Rehema <br> Glad to see you !</h1>
             
         
     </div>
+<section id="portfolio" class="work-section">
+    <section class="card card work">
```

Comment:
- [ ] There you did the opposite, please kindly change `section` tags of all 4 work cards to just `div` element 🙌🏼. 
- [ ] And there is no need to add same 2 card classes to one element

```html
<section class="card card work">
```